examples/llm/glem.py

```diff
@@ -79,7 +79,7 @@ def main(args):
 
     dataset = PygNodePropPredDataset(f'ogbn-{dataset_name}', root=root)
     split_idx = dataset.get_idx_split()
-    data = dataset.data
+    data = dataset._data
```

Comment:
It's usually a good idea to avoid exposing private attributes to users.
```suggestion
    data = dataset[0]
```

Reply:
Done. Thanks for the advice!